sites/friday/src/get-started/django/deploy/configure.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Configure infrastructure
+subtitle: Deploying {{% param "framework" %}} to {{% vendor/name %}}
+weight: -16
+description: Configure infrastructure for {{% param "framework" %}} in a single YAML file. 
+framework: Django
+---
+<!-- {{% guides/steps ordered="true" root="/get-started/express/deploy" %}} -->
+
+<!-- Status of the integration (failure) -->
+{{% guides/deploy/configure/status %}}
+
+<!-- Project:init -->
+{{% guides/deploy/configure/project-init "django" %}}
+
+- Choose **Python** for the language.
+- Give the project a name, something like `{{% vendor/cli %}}_django`.
+- Choose any combination of services you need for your project. If you are working with the project skeleton (selected the **Start from Scratch**) option, a common choice is **PostgreSQL** and **Redis**.
+
+This command will generate two files for you:
+
+- A `.environment` file, where you can save environment variables that will be sourced into the environment.
+- A `{{% vendor/configfile "app" %}}` file - the main {{% vendor/name %}} configuration file.
+
+1. **(Optional) Configure root**. The configuration generated assumes that the Django project root is the repository root.
+If this is not the case, and your Django code is nested within a directory, add the following `source.root` configuration to `{{% vendor/configfile "app" %}}`.
+
+    ```yaml {configFile="app"}
+    applications:
+        APP_NAME:
+            # ...GENERATED CONFIGURATION...
+
+            source:
+                root: NESTED_DIR
+
+            # ...GENERATED CONFIGURATION...
+    ```
+
+1. **Configure the build and deploy**. The `{{% vendor/configfile "app" %}}` contains a `hooks` block, containing configuration for how Django should be built and deployed. 
+The configuration for the "From scratch" app is shown below.
+Update your own build and deploy steps using it as an example.
+
+    ```yaml {configFile="app"}
+    applications:
+        APP_NAME:
+            # ...GENERATED CONFIGURATION...
+
+            hooks:
+                build: |
+                    set -eux
```

Comment:
The indentation you have here is REALLY indented: 16 spaces. If copied & pasted into what the `project:init` command generates, it looks... off. 
<img width="507" alt="image" src="https://github.com/platformsh/platformsh-docs/assets/2332956/24199c99-76c9-4980-9488-641cf25b29e0">